### PR TITLE
Add dashboard for Network

### PR DESCRIPTION
**Pull request by monicasarbu**
This PR create a dedicated Kibana dashboard for the System Network data.

### CODE REVIEW

**Comment by ruflin** [comment]
Would it be possible that we add a screenshot to the docs for each dashboard we have? That will make it very easy visible for users to see what they get and makes our docs more beautiful. Disadvantage is that we must keep the screenshots up-to-date ...

**Comment by ruflin** [comment]
All dashboards must be added under the specific module in `_meta/kibana`. Otherwise when you run `make collect` it will overwrite all your dashboards. It should be as easy as just copy the files over.

**Comment by monicasarbu** [comment]
Yes, I am planning to add screenshots of the dashboards later after I finish creating a dashboard for each Metricbeat module. I hope to write docs about how to use the dashboards.

**Comment by monicasarbu** [comment]
After internal discussions, we decided to create dashboards per Metricbeat module and not per entire Metricbeat and have the navigation bar only for the system module. You can access the dashboards of the other Metricbeat modules by clicking on`Open` in Kibana.

File: metricbeat/include/list.go [hunk 1]
```diff
@@ -25,6 +25,7 @@ import (
 	_ "github.com/elastic/beats/metricbeat/module/system/diskio"
 	_ "github.com/elastic/beats/metricbeat/module/system/filesystem"
 	_ "github.com/elastic/beats/metricbeat/module/system/fsstat"
+	_ "github.com/elastic/beats/metricbeat/module/system/load"
```
Comment (ruflin) [inline metricbeat/include/list.go]: why was this changed? switch forth / back between branches? `make collect` should clean it up.

Comment (monicasarbu) [inline metricbeat/include/list.go]: Not sure how it ended there. I removed it manually.

Comment (monicasarbu) [inline metricbeat/include/list.go]: Ah, yes. `make collect

Comment (ruflin) [inline metricbeat/include/list.go]: @monicasarbu Probably the empty directories for load still exists from your other branch ;-)
